tests/phpunit/integration/Core/Storage/User_OptionsTest.php

```diff
@@ -120,17 +120,6 @@ public function test_network_mode_delete() {
 		$this->assertFalse( metadata_exists( 'user', $user_id, 'test-key' ) );
 	}
 
-	public function test_user_input_state_value() {
```

Comment:
Same here, why did you remove it? Instead of removing this test, we need to create an instance of the `User_Input_State` class and register it.

Reply:
This was a misspelled test [Remove misplaced test.](https://github.com/google/site-kit-wp/pull/6366/commits/82e87b333e3fd8581a47135a7b74dc52be3f9b6f)

Reply:
Ok, thanks.